src/pkg/filters/filters.go

```diff
@@ -133,9 +163,102 @@ func NotPrefix(target string) Filter {
 	return newFilter(TargetPrefixes, target, true)
 }
 
+// PathPrefix creates a filter where Compare(v) is true if
+// target.Prefix(v) &&
+// split(target)[i].Equals(split(v)[i]) for _all_ i in 0..len(target)-1
+// ex: target "/foo/bar" returns true for input "/foo/bar/baz",
+// but false for "/foo/barbaz"
+//
+// Unlike single-target filters, this filter accepts a
+// slice of targets, and implicitly handles comparisons as
+// an `any-match` check (ie: Compare will return true if any
```

Comment:
nit: maybe be less confusing to use `logical OR` instead of `any-match` since `any` is often used for `*` in selectors

Reply:
Fair point, fixed up the language a little.